auctions/README.md

```diff
@@ -0,0 +1,117 @@
+# Auctions
+
+## Auction object
+
+```
+{
+  'id': <auction-id>,
+  'seller': <userId>,
+  'item_type': <example: neelam>,
+  'initial_price': <example: 100 dinero>,
```

Comment:
We should also include the quantity of the item being sold

Reply:
Yeah, how could we forget that 😆